src/System.Data.SqlClient/ref/System.Data.SqlClient.Manual.cs

```diff
@@ -7,6 +7,25 @@
 
 namespace System.Data.SqlClient
 {
+    // Declaring members from stripped base class CollectionBase
+    public sealed partial class SqlBulkCopyColumnMappingCollection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
```

Comment:
With `CollectionBase` the property `int Capacity` was available as well. With the implementation of the interfaces, the property cannot be used by the clients. 
This will break any clients using the property.